### PR TITLE
Extend language support

### PR DESCRIPTION
**Pull request by cgwalters**
See https://bugzilla.redhat.com/show_bug.cgi?id=1186757

Note this requires https://github.com/projectatomic/rpm-ostree/pull/525

### CODE REVIEW

File: treecompose-post.sh [hunk 1]
```diff
@@ -5,9 +5,45 @@ set -e
...
+hi_IN
+kn_IN
+ml_IN
+mr_IN
+or_IN
+pa_IN
+ta_IN
+te_IN
+"
+
```
Comment (dustymabe) [inline treecompose-post.sh]: a comment or two couldn't hurt

**Comment by cgwalters** [comment]
OK, done :arrow_up:   But most of the information is in the issues, wiki pages, and source code.

**Comment by jlebon** [comment]
OK, that works. 👍
Does the tree composer automatically use the rpm-ostree from git to compose trees?

**Comment by cgwalters** [comment]
Depends which one you mean.  For CAHC, it uses atomic7-testing which so far I've been using stable releases.  For this case I think I'll just cherry pick the patch.

**Comment by cgwalters** [comment]
OK, this is rebased now on top of the Fedora commit (which I just pushed to master).

**Comment by jlebon** [comment]
LGTM as always, and I see that rpm-ostree v2016.3 is in the repo, so this looks ready to merge.

**Pull request merged**